### PR TITLE
[Cloudflare for SaaS] Fixed Early Hints API call

### PR DESCRIPTION
**Pull request by mmalden**

### CODE REVIEW

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 1]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
+3. If you would like to enable Early Hints for a preexisting custom hostname, locate your custom hostname ID by making a ‘get’ call in the API:
 
 ```json
 curl -X GET "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames" \
     -H "X-Auth-Email: {email}" \
     -H "X-Auth-Key: {key}" \
     -H "Content-Type: application/JSON"
+```
+
+If you are [creating a new custom hostname](https://api.cloudflare.com/#custom-hostname-for-a-zone-create-custom-hostname), make an API call such as the example below, specifying `"early_hints":"on"`:
```
Comment (WalshyDev) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
If you are [creating a new custom hostname](https://api.cloudflare.com/#custom-hostname-for-a-zone-create-custom-hostname), make an API call such as the example below, specifying `"early_hints": "on"`:
```

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 2]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
 
+```json
+curl -X POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames" \
+     -H "X-Auth-Email: {email}" \
+     -H "X-Auth-Key: {key}" \
+     -H "Content-Type: application/JSON" \
+        --data '{"hostname":"{your_custom_hostname}"}, "ssl”:{“wildcard”:”true”,”early_hints":"on"}}'
```
Comment (WalshyDev) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
        --data '{"hostname":"{your_custom_hostname}"}, "ssl”:{"wildcard": "true", "early_hints": "on"}}'
```

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 3]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
 ```
 
-4. Make an API call such as the example below, specifying `”early_hints”:”on”` within the [custom metadata](/cloudflare-for-saas/ssl/hostname-specific-behavior/custom-metadata/):
+4. For an existing custom hostname, an API call such as the example below, specifying `”early_hints”:”on”`:
```
Comment (WalshyDev) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
4. For an existing custom hostname, an API call such as the example below, specifying `"early_hints2: "on"`:
```

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 4]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
+--data '{
  "ssl": {
-   "method": "http",
-   "type":"dv"
- },
- "custom_metadata": {
-   "customer_id": "12345",
-   "early_hints”:”on”
+   "wildcard":"true",
+   "early_hints:"on"
```
Comment (WalshyDev) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
   "early_hints": "on"
```

**Review by WalshyDev — CHANGES_REQUESTED** [review]
Macos smart quotes

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 5]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
 
 2. Locate your Authentication Key by selecting **My Profile** > **API tokens** > **Global API Key**.
 
-3. Locate your custom hostname ID by making a ‘get’ call in the API:
+3. If you would like to enable Early Hints for a preexisting custom hostname, locate your custom hostname ID by making a ‘get’ call in the API:
```
Comment (pedrosousa) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
3. If you would like to enable Early Hints for a preexisting custom hostname, locate your custom hostname ID by making a `GET` request to the API:
```

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 6]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
 
 ```json
 curl -X GET "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames" \
     -H "X-Auth-Email: {email}" \
     -H "X-Auth-Key: {key}" \
     -H "Content-Type: application/JSON"
```
Comment (pedrosousa) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
    -H "Content-Type: application/json"
```
This content type is in lower case (I checked the [RFC](https://www.rfc-editor.org/rfc/rfc8259.html#section-11)).

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 7]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
+```
+
+If you are [creating a new custom hostname](https://api.cloudflare.com/#custom-hostname-for-a-zone-create-custom-hostname), make an API call such as the example below, specifying `"early_hints":"on"`:
 
+```json
+curl -X POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames" \
+     -H "X-Auth-Email: {email}" \
+     -H "X-Auth-Key: {key}" \
+     -H "Content-Type: application/JSON" \
```
Comment (pedrosousa) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
     -H "Content-Type: application/json" \
```

**Review by pedrosousa — APPROVED** [review]

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md (hunk 1 above)
Comment (aanfath) [reply content/cloudflare-for-saas/related-products/early-hints-for-saas.md]: #SNEP.. 
Screenshot_2022-07-19-14-32-06-703_com google android gm

My ambition of of learning from/through Cloud fare is expressed/informed and it reached in their two email. (Hr@cloudflare.com, careers@cloudflare.com) 

They are not responding.. Why? 

There are many things to learn from/through cloud fare and I am wait for to start from/through them. 

Successful saving of nature earth planets.. 

Anand Vayal Veettil😊
00919747344544

Dears, 

Regards...

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 8]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
+```json
+curl -X POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames" \
+     -H "X-Auth-Email: {email}" \
+     -H "X-Auth-Key: {key}" \
+     -H "Content-Type: application/json" \
+        --data '{"hostname":"{your_custom_hostname}"}, "ssl”:{"wildcard": "true", "early_hints": "on"}}'
 ```
 
-4. Make an API call such as the example below, specifying `”early_hints”:”on”` within the [custom metadata](/cloudflare-for-saas/ssl/hostname-specific-behavior/custom-metadata/):
+4. For an existing custom hostname, an API call such as the example below, specifying `"early_hints2: "on"`:
```
Comment (WalshyDev) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
4. For an existing custom hostname, an API call such as the example below, specifying `"early_hints": "on"`:
```

File: content/cloudflare-for-saas/related-products/early-hints-for-saas.md [hunk 9]
```diff
@@ -22,33 +22,38 @@ Before you can employ Early Hints for SaaS, you need to create a custom hostname
...
 
 ```json
 $ curl -sXPATCH \
 
-"https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/custom_hostnames/<HOSTNAME_ID>" \
+"https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames/{hostname_id}}" \
```
Comment (WalshyDev) [inline content/cloudflare-for-saas/related-products/early-hints-for-saas.md]:
```suggestion
"https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames/{hostname_id}" \
```

**Review by WalshyDev — CHANGES_REQUESTED** [review]
2 other small things :)

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**